docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+* Er bestaat in de ZRC geen rechtstreekse relatie tussen het merendeel van de domein/objectgegevens en een zaak (dit is beperkt tot zaakobjecten en zaaktypespecifieke gegevens). Idem voor gedetailleerde procesgegevens en een zaak. M.a.w. in het geval van erg complexe processen, kent de zaak in de ZRC enkel dit proces op hoofdlijnen (de zaakstatussen).
+* Wel liggen deze relaties in de specifieke zaakafhandelcomponent: daarin is (impliciet of expliciet) een proces geconfigureerd (=procesdefinitie (“procestype”), bijvoorbeeld afhandelen evenementensubsidieaanvraag). Het verloop van een specifieke instantie van dat proces volgt dit procestype. En er liggen relaties tussen dit proces en de domein/objectregistratie(s) die bij dit proces horen.
+* De ZAC heeft wel uiteraard weet van welk zaaktype en welke zaak er bij een bepaald procestype en bij een bepaalde procesinstantie horen, maar vanuit een zaak is dit dus niet direct opvraagbaar. Opmerking AK De specifieke ZAC, ja.. Maar de generieke ZAC toch niet?
+
+### 1.3.3. Situatie in een Gegevenslandschap
+
+In een gegevenslandschap komt bovenstaand plaatje er, door het scheiden van proces en data, eenmalige opslag en bevraging ‘bij de bron’ en het verminderen van informatiestromen anders uit te zien (zie figuur 2):
+
+![zgw-in-gegevenslandschap.png](./bestanden/zgw-in-gegevenslandschap.png?raw=true)
+*Figuur 2 - ZGW in een Gegevenslandschap*
+
+* De domein/objectgegevens gegevens zijn uit de Zaakafhandelcomponent verdwenen. Procesdefinities en procesgegevens blijven daar echter achter .
+* Uitgangspunt in het gegevenslandschap is: alle gegevens worden op één plaats opgeslagen. En alle soortgelijke gegevens worden bij elkaar opgeslagen. Er komt dus een registratie met besluiten, een registratie met aanvragen en meldingen, een registratie met klanten, een registratie medewerkers. Etc.
+* De “Zaak” is een belangrijk verbindend element tussen de objecten in de registraties. Getekend: een zaak aggregeert al deze andere objecten. Zaak fungeert zo als “sleutel” in een grote “verwijsindex” naar objecten in andere registraties.  Opmerking AK En wat moet ik me daarbij voorstellen, wat is het nut daarvan? Welke op hun beurt weer kunnen relateren naar concepten in andere registraties, bv. de BRP of het HR. Opmerking AK Er zijn ook andere verbindende objecten zoals de klant (of een NP of NNP) die een mail gestuurd heeft (document), houder is van een vergunning, een handhavingsverzoek heeft ingediend (aanvraag/melding) en een paspoort heeft gekregen. Dergelijke relaties zijn in de figuur niet gevisualiseerd. Terecht, de focus is zaakgericht werken. Maar wel even vermelden.
+* Documenten die bij een zaak horen kunnen dan in de DRC worden opgehaald mbv het zaaknummer. Idem voor besluit(en) per zaak en betrokkenen.
```

Comment:
Hier is nog een discussie over in naar aanleiding van #78, waarna volgens mij in #166 de discussie verder ging.

Reply:
oh, was daar discussie over :-)
ik ga #166 etc verwerken in een update van het document